app/bundles/ReportBundle/Views/Report/form.html.php

```diff
@@ -136,7 +141,54 @@
                                 </div>
                             </div>
                         </div>
-
+                        <div class="tab-pane fade bdr-w-0" id="schedule-container">
+                            <div class="row">
+                                <div class="col-md-12">
+                                    <div class="pa-md">
+                                        <div class="checkbox">
+                                            <?php echo $view['form']->row($form['isScheduled']); ?>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div id="schedule_form">
+                                <div class="row">
+                                    <div class="col-md-6">
+                                        <div class="pa-md">
+                                            <?php echo $view['form']->row($form['toAddress']); ?>
+                                        </div>
+                                    </div>
+                                </div>
+                                <div class="row" id="schedule_form">
+                                    <div class="col-md-6">
+                                        <div class="pa-md">
+                                            <?php echo $view['form']->row($form['scheduleUnit']); ?>
```

Comment:
When I change the value of this, the AJAX request `/s/reports/schedule/preview/1/MONTHLY/MO/1` fatals with ```Symfony\Component\Debug\Exception\FatalThrowableError: Class 'Recurr\Rule' not found - in file app/bundles/ReportBundle/Scheduler/Builder/SchedulerBuilder.php```

I get this same error when I try to save the report.

Reply:
Never mind on this. I neglected to run composer install. Retesting.